cmake/Install.cmake

```diff
@@ -113,6 +113,9 @@ INSTALL(FILES share/config.fish
 # -$v $(INSTALL) -m 755 -d $(DESTDIR)$(extra_completionsdir)
 # -$v $(INSTALL) -m 755 -d $(DESTDIR)$(extra_functionsdir)
 # -$v $(INSTALL) -m 755 -d $(DESTDIR)$(extra_confdir)
```

Comment:
I think it's time to remove these comments. Doesn't have to be in this PR, but I think we're truly past make now.